### DISCUSSION TITLE
Is there a way to list all projects in a dataview table or even in a new project?

### DISCUSSION THREAD

**Discussion by fmj79** [marcusolsson/obsidian-projects#475] [Help] [answered]
Hi, I'd like to know if there is any way to list all my current project in the same page with active links. It could be with dataview, projects plugin or otherwise. I'd like to list them in my landing page and click to go directly to the project I want to add new info to and avoid clicking at the projects icon and, then, on the dropdown list. Thanks a lot

**Comment by marcusolsson** [comment, marked as answer]
While there's no way to programmatically list all projects, you can manually create links to projects and views in your notes.

For more information, check out #377.
